webpack.config.dev.js

```diff
@@ -5,7 +5,7 @@ exports.entry.unshift('webpack/hot/dev-server');
 exports.module.loaders[0].loaders = ['babel', 'react-hot', 'jsx?harmony'];
 exports.plugins[1] = new webpack.DefinePlugin({
   __PROD__: false,
-  __S3_BUCKET__: 'landline'
+  __S3_BUCKET__: JSON.stringify('landline')
```

Comment:
But is there any reason for the api url being passed in when the host
initializes it? In other words, is there any reason for the url to be set
at runtime rather than compile time?

On Mon, Mar 16, 2015 at 6:52 PM Charles Pletcher <notifications@github.com>
wrote:

> In webpack.config.dev.js
> <https://github.com/asm-products/landline-web/pull/5#discussion_r26509964>
> :
>
> > @@ -5,7 +5,7 @@ exports.entry.unshift('webpack/hot/dev-server');
> >  exports.module.loaders[0].loaders = ['babel', 'react-hot', 'jsx?harmony'];
> >  exports.plugins[1] = new webpack.DefinePlugin({
> >    __PROD__: false,
> > -  __S3_BUCKET__: 'landline'
> > +  __S3_BUCKET__: JSON.stringify('landline')
>
> We can't do that, though, because Landline doesn't know what the API URL
> is before its host initializes it (you can see this happening in
> index.html).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asm-products/landline-web/pull/5/files#r26509964>.
>